### PR TITLE
Rename isQuery to committed in vochain

### PR DESCRIPTION
**Pull request by G10h4ck**
Added also some documentation about it's usage on all methods that sports this param

### CODE REVIEW

**Comment by G10h4ck** [comment]
I have got a few doubts about the documentation I have written, flagged those with TODO, may someone with more insight than me take a look?

File: vochain/state.go [hunk 1]
```diff
@@ -658,13 +667,17 @@ func (v *State) RemoveValidator(address []byte) error {
 }
 
 // Validators returns a list of the validators saved on persistent storage
-func (v *State) Validators(isQuery bool) ([]*models.Validator, error) {
-	if !isQuery {
+// TODO: do 'saved on persistent storage' make sense also when committed == false?
```
Comment (jordipainan) [inline vochain/state.go]: Nope, in this case and following the other methods I think we should rewrite the comment.
I propose to change it to: `Validators returns a list of the chain validators`

File: vochain/state.go [hunk 2]
```diff
@@ -881,14 +903,18 @@ func (v *State) EnvelopeExists(processID, nullifier []byte, isQuery bool) (bool,
 }
 
 // iterateVotes iterates fn over state tree entries with the processID prefix.
-// if isQuery, the IAVL tree is used, otherwise the AVL tree is used.
+// if committed is true, the IAVL tree is used, otherwise the AVL tree is used.
+// TODO: does previous line is more descriptive then the followings 3 ?
```
Comment (jordipainan) [inline vochain/state.go]: If you're not aware that `I` stands for Immutable then sadly not ... So the 3 following lines are better and maintain consistency with the other methods.

File: vochain/state.go [hunk 3]
```diff
@@ -658,13 +667,17 @@ func (v *State) RemoveValidator(address []byte) error {
 }
 
 // Validators returns a list of the validators saved on persistent storage
```
Comment (jordipainan) [inline vochain/state.go]:
```suggestion
// Validators returns a list of the chain validators
```

File: vochain/state.go [hunk 4]
```diff
@@ -881,14 +903,18 @@ func (v *State) EnvelopeExists(processID, nullifier []byte, isQuery bool) (bool,
 }
 
 // iterateVotes iterates fn over state tree entries with the processID prefix.
-// if isQuery, the IAVL tree is used, otherwise the AVL tree is used.
+// if committed is true, the IAVL tree is used, otherwise the AVL tree is used.
```
Comment (jordipainan) [inline vochain/state.go:906]:
```suggestion
```

File: vochain/state.go (hunk 2 above)
Comment (randomshinichi) [reply vochain/state.go]: And what does AVL stand for?

Comment (jordipainan) [reply vochain/state.go]: https://pkg.go.dev/github.com/tendermint/iavl

**Comment by jordipainan** [comment]
Missing changes on https://github.com/G10h4ck/vocdoni-node/blob/mv_isquery_committed/vochain/process.go#L116 & https://github.com/G10h4ck/vocdoni-node/blob/mv_isquery_committed/vochain/process.go#L125

With these remaining lines I think we can say we got rid of IsQuery naming :)

**Comment by G10h4ck** [comment]
> Missing changes on https://github.com/G10h4ck/vocdoni-node/blob/mv_isquery_committed/vochain/process.go#L116 & https://github.com/G10h4ck/vocdoni-node/blob/mv_isquery_committed/vochain/process.go#L125
> 
> With these remaining lines I think we can say we got rid of IsQuery naming :)

I forgot to save process.go before committing ;-)

**Review by jordipainan — APPROVED** [review]

**Comment by randomshinichi** [comment]
Thanks for taking on this issue, code feels better now!

**Comment by jordipainan** [comment]
self-reminder: Squash

**Pull request merged**